app/assets/scss/_publication.scss

```diff
@@ -1,383 +1,4 @@
-        border-spacing: 0;
-        margin-bottom: $gutter*2;
-        th, td {
-          @include core-14;
-          background-color: transparent;
-          vertical-align: top;
-          text-align: left;
-          padding: $gutter-one-third $gutter-two-thirds $gutter-one-third 0;
-          border-bottom: 1px solid $grey-2;
-          border-width: 0 0 1px 0;
-        }
-        th {
-          font-weight: bold;
-        }
-        td {
-          p, ul, ol {
-            margin: 0;
-          }
-          p+p, p+ul, p+ol,
-          ul+p, ul+ol,
-          ol+p {
-            margin-top: $gutter-one-third;
-          }
-        }
-      }
-      .contact {
-        @extend %contain-floats;
-        background: $panel-colour;
-        margin: $gutter $gutter-half*-1;
-        padding: $gutter-one-third $gutter-half;
-        position: relative;
-
-        h4 {
-          @include core-19;
-          font-weight: bold;
-          margin: $gutter-one-third 0 0;
-        }
-        p {
-          @include copy-19;
-          margin: 0;
-        }
-        .adr,
-        .email-url-number,
-        .comments {
-          @include media(tablet) {
-            width: $half;
-            float: left;
-          }
-        }
-        .email-url-number {
-          p {
-            margin: 0;
-          }
-          span {
-            display: block;
-          }
-        }
-        .comments {
-          @include core-16;
-        }
-      }
-      .call-to-action {
-        margin: $gutter 0 $gutter;
-        background: $panel-colour none no-repeat 98% $gutter-two-thirds;
-        padding: $gutter-one-third ($gutter*2) $gutter-one-third $gutter-half;
-        position: relative;
-
-        h4 {
-          margin-top: 0;
-        }
-        p {
-          margin: 0;
-          padding: 0;
-        }
-        ul, ol {
-          margin-left: $gutter-two-thirds;
-          list-style-position: outside;
-        }
-        h3 {
-          padding: $gutter-one-sixth 0;
-          margin-top: 0px;
-          .number {
-            background: none;
-            @include media(tablet) {
-              width: $gutter-one-third*9;
-              margin-left: $gutter-one-third*-9;
-            }
-          }
-        }
-      }
-
-      .fraction {
-        sup,
-        sub {
-          @include core-14;
-        }
-        img {
-          display: inline-block;
-          width: auto;
-          margin: 0 0 -5px 0;
-        }
-      }
-    }
-  }
-  .back-to-content {
-    position: fixed;
-    bottom: $gutter-half;
-    a {
-      display: block;
-
-      &:before {
-        content: "\2191 ";
-      }
-    }
-  }
-  .js-back-to-content-stop {
-    clear: both;
-  }
-  p.report-a-problem-toggle a {
-    @include core-16;
-    color: $secondary-text-colour;
-    text-decoration: underline;
-  }
-  .report-a-problem-container {
-    margin-left: 0;
-  }
-
-}
-
-.floated-children {
-  overflow: hidden;
-}
 
 .terms-and-conditions {
```

Comment:
So I think we have the same issue with the cookies template for the terms and conditions template - we no longer need it as it is provided by the buyer FE.

Therefore I think we should delete the terms and conditions template as part of this PR. 

With that being the case, there is only one other use of the `terms-and-conditions` class which is on the link to the terms and conditions page in the footer template. Looking at this particular styling, it does not apply to it (I think we dealt with the same situation on one of the other apps). Therefore we should also be able to remove the whole publication.scss file.

Reply:
Done